### ISSUE TITLE
None is not a valid permission (enum) preventing new permissions code from working properly

### ISSUE THREAD

**Issue by vorpal-buildbot** [silasary/Penny-Dreadful-Discord-Bot#12081]
Reported on Discord by @bakert

**Comment by bakert** [comment]
It's giving affected folks a 500 that looks like this:

```   
return set(Permission(v) for v in db().values(sql, [discord_id]))
  File "/usr/lib/python3.10/enum.py", line 385, in __call__
    return cls.__new__(cls, value)
  File "/usr/lib/python3.10/enum.py", line 710, in __new__
    raise ve_exc
ValueError: None is not a valid Permission
```

**Comment by bakert** [comment]
Fixed in #12082
